test/unit/services/policies/policies_list_service_test.rb

```diff
@@ -23,16 +25,43 @@ def setup
   }.freeze
 
   test 'call with error response on gateway side' do
-    stub_request(:get, "https://apicast-staging.proda.example.com/policies")
+    stub_request(:get, APICAST_REGISTRY_URL)
       .to_return(status: 502, body: '{"error":"A server error occured"}',
                  headers: { 'Content-Type' => 'application/json' })
 
     account = FactoryBot.build_stubbed(:simple_provider)
-    assert_nil @service.call(account)
+
+    assert_nothing_raised do
+      policies = @service.call(account)
+      assert_nil policies
+    end
+  end
+
+  test 'call with no apicast registry url' do
+    ThreeScale.config.sandbox_proxy.expects(:apicast_registry_url).returns(nil)
+    JSONClient.expects(:get).with(nil).raises(SocketError)
+
+    account = FactoryBot.build_stubbed(:simple_provider)
+
+    assert_nothing_raised do
```

Comment:
This does nothing 😝 So imo it is better without the `assert_nothing_raised`. You are already asserting that the response is `nil` so that also implies that nothing is raised 😄